#use wml::debian::translation-check translation="1.1"
<define-tag description>PRNG が状況により正常に乱数の種を再生成しない</define-tag>
<define-tag moreinfo>
<p>python-crypto の疑似乱数生成器に暗号の脆弱性が発見されました。</p>

<p>同一の親プロセスから複数のプロセスが分岐 (fork) している場合、
状況によって競合状態のため乱数生成器による乱数の種の再生成が妨害されます。
これは全プロセスで同一の出力を生成することにつながり、
暗号の鍵のような機密の値を漏洩する可能性があります。</p>

<p>旧安定版 (oldstable) ディストリビューション (squeeze)
では、この問題はバージョン 2.1.0-2+squeeze2 で修正されています。</p>

<p>安定版 (stable) ディストリビューション (wheezy)
では、この問題はバージョン 2.6-4+deb7u3 で修正されています。</p>

<p>テスト版 (testing) ディストリビューション (jessie)
では、この問題はバージョン 2.6.1-2 で修正されています。</p>

<p>不安定版 (unstable) ディストリビューション (sid)
では、この問題はバージョン 2.6.1-1 で修正されています。</p>

<p>直ちに python-crypto パッケージをアップグレードすることを勧めます。</p>
</define-tag>

# do not modify the following line
#include "$(ENGLISHDIR)/security/2013/dsa-2781.data"

